hypervisor/arch/x86/configs/cb2_dnv.config

```diff
@@ -0,0 +1,7 @@
+CONFIG_BOARD="cb2_dnv"
```

Comment:
It would have been better to follow the same (albeit implicit) convention here and use `dnv-cb2`.

Reply:
OK, we will discuss, and it this makes sense, we will change it